test/test_serialize_explanation.py

```diff
@@ -88,14 +100,28 @@ def _assert_explanation_equivalence(actual, expected):
             assert actual_value == expected_value
 
 
+def _assert_numpy_explanation_types(actual, expected):
+    # assert "_" variables equivalence
+    if hasattr(actual, PrivateExplainParams.U_LOCAL_IMPORTANCE_VALUES):
+        assert(isinstance(actual._local_importance_values, np.ndarray))
+        assert(isinstance(expected._local_importance_values, np.ndarray))
+        np.testing.assert_array_equal(actual._local_importance_values, expected._local_importance_values)
+    if hasattr(actual, PrivateExplainParams.U_EVAL_DATA):
+        assert(isinstance(actual._eval_data, np.ndarray))
+        assert(isinstance(expected._eval_data, np.ndarray))
+        np.testing.assert_array_equal(actual._eval_data, expected._eval_data)
+
+
 # performs serialization and de-serialization for any explanation
 # tests to verify that the de-serialized result is equivalent to the original
 # exposed outside this module to allow any test involving an explanation to
 # incorporate serialization testing
-def verify_serialization(explanation, extra_path=None, exist_ok=False):
+def verify_serialization(explanation, extra_path=None, exist_ok=False, assert_types=False):
     path = 'brand/new/path'
```

Comment:
maybe assert_types can be renamed to assert_numpy_types.

Reply:
done!